tests/unit/plugins/v1/test_catkin.py

```diff
@@ -1909,7 +1912,7 @@ def test_find_system_dependencies_system_only(self):
 
         self.assertThat(
             catkin._find_system_dependencies(
-                {"foo"}, self.rosdep_mock, self.catkin_mock
+                {"foo"}, self.rosdep_mock, self.catkin_mock, set([])
```

Comment:
No need for the cast, you can just use `set()` directly (this applies throughout the tests):

```suggestion
                {"foo"}, self.rosdep_mock, self.catkin_mock, set()
```